atlassian_jwt_auth/contrib/tests/test_flask.py

```diff
@@ -0,0 +1,110 @@
+import unittest
+
+from flask import Flask
+from mock import patch
+
+import atlassian_jwt_auth
+from atlassian_jwt_auth.tests import utils
+from atlassian_jwt_auth.contrib.flask_app import requires_asap
+from atlassian_jwt_auth.contrib.tests.utils import static_verifier
+
+
+def get_app():
+    app = Flask(__name__)
+    app.config.asap = {
+        'VALID_AUDIENCE': 'server-app',
+        'VALID_ISSUERS': ['client-app']
+    }
+
+    @app.route("/")
+    @requires_asap
+    def view():
+        return "OK"
+
+    return app
+
+
+def create_token(issuer, audience, key_id, private_key):
+    signer = atlassian_jwt_auth.create_signer(
+        issuer, key_id, private_key
+    )
+    return signer.generate_jwt(audience)
+
+
+class FlaskTests(utils.RS256KeyTestMixin, unittest.TestCase):
```

Comment:
I am not accepting tests that do not have test cases that cover using the `RS256KeyTestMixin` and the `ES256KeyTestMixin` any more. Feel free to have a base test class and sub-classes that use the various mixins.